R/runs.R

```diff
@@ -672,6 +678,49 @@ log_artifact <- function(x, FUN = saveRDS, filename, run_id, ...) {
   s3_file
 }
 
+#' @importFrom aws.s3 put_object
+#' @rdname log_artifact
+#' @export
+log_artifact.ggplot <- function(x, FUN, filename, run_id, ...) {
```

Comment:
`FUN` gets completely ignored in favor of forcing the usage of `ggplot2::ggsave`. it might be worth warning the user about this. I've chosen to do things like this in the case that the user provides a different saving function, which would likely fail